src/hotspot/share/compiler/compileBroker.cpp

```diff
@@ -906,7 +906,6 @@ JavaThread* CompileBroker::make_thread(ThreadType type, jobject thread_handle, C
     // queue compile jobs which will run at whatever default priority the
     // newly created CompilerThread runs at.
 
-
```

Comment:
We don't have this hunk in the original changeset?

Reply:
Sorry, no reason for this.